ExceptionHandler.cs

```diff
@@ -0,0 +1,81 @@
+namespace CarApplication
+{
+    public class ExceptionHandler 
```

Comment:
ExceptionHandler is more like a generalized name. Rename it to something more specific.

Reply:
Resolevd